java/src/main/java/com/factset/protobuf/stach/extensions/v2/ColumnOrganizedStachExtension.java

```diff
@@ -103,6 +104,18 @@ private TableData generateTable(PackageProto.Package packageObj, String primaryT
             table.getMetadata().put(entry.getKey(), StachUtilities.valueToString(entry.getValue().getValue()));
         }
 
+        table.getMetadata().forEach((k,v) -> {
+            List<String> values = new ArrayList<>();
+
+            String val = v.substring(1,v.length()-1); // remove brackets
+            String[] valArray = val.split(","); // if multiple items, split on ","
```

Comment:
won't this split on commas within an array item as well?

Reply:
In my latest commit I updated from returning `Map<String, List<String>>` to `Map<String, List<Value>>` , which also fixed this bug